.gitignore

```diff
@@ -1,5 +1,6 @@
 build
 build-vagrant
+buildxc
```

Comment:
I know this is for xcode build via cmake, but the cmake lines for that are not in this PR, so should just be dropped.